client/route.js

```diff
@@ -11,6 +11,10 @@ Route = function(router, path, options) {
   this._middlewares = options.middlewares || [];
   this._subsMap = {};
   this._router = router;
+
+  if (group) {
```

Comment:
Always try to define group here. Don't use if. (like `this.group = group`)
Then V8 can optimize it.
(Not much critical here since we don't have heavy use of this class, but still good to consider)